docs/src/main/paradox/couchbase.md

```diff
@@ -20,164 +30,117 @@ The table below shows direct dependencies of this module and the second tab show
 
 @@dependencies { projectId="couchbase" }
 
+# Overview
```

Comment:
Above in the `@@dependency` change `$scalaBinaryVersion$` to `$scala.binary.version$` and `$version$` to `$project.version$`.